engine/security/trust/trust_sandbox.md

```diff
@@ -61,150 +62,181 @@ and can be destroyed after you destroy the container.
-        $ cd trustsandbox
-
-2. Create a file called `docker-compose.yml` with your favorite editor.  For example, using vim:
-
-        $ touch docker-compose.yml
-        $ vim docker-compose.yml
-
-3. Add the following to the new file.
-
-        version: "2"
-        services:
-          notaryserver:
-            image: dockersecurity/notary_autobuilds:server-v0.5.1
-            volumes:
-              - notarycerts:/var/lib/notary/fixtures
-            networks:
-              - sandbox
-            environment:
-              - NOTARY_SERVER_STORAGE_TYPE=memory
-              - NOTARY_SERVER_TRUST_SERVICE_TYPE=local
-          sandboxregistry:
-            image: registry:2.4.1
-            networks:
-              - sandbox
-            container_name: sandboxregistry
-          trustsandbox:
-            image: docker:dind
-            networks:
-              - sandbox
-            volumes:
-              - notarycerts:/notarycerts
-            privileged: true
-            container_name: trustsandbox
-            entrypoint: ""
-            command: |-
-                sh -c '
-                    cp /notarycerts/root-ca.crt /usr/local/share/ca-certificates/root-ca.crt &&
-                    update-ca-certificates &&
-                    dockerd-entrypoint.sh --insecure-registry sandboxregistry:5000'
+    ```console
+    $ mkdir trustsandbox
+    $ cd trustsandbox
+    ```
+
+2.  Clone the official `notary` repository:
+
+    ```console
+    $ git clone https://github.com/notaryproject/notary.git
+    ```
+
+3.  Create a file called `docker-compose.yml` with your favorite editor and add the following to the new file:
+
+    ```yaml
+    services:
+      notaryserver:
+        image: notary:server-0.7.0
+        networks:
+          - sandbox
+        depends_on:
+          - notarysigner
+        volumes:
+          - ./notary:/notarydir
+        command: |-
+          sh -c "/notarydir/migrations/migrate.sh &&
+          cd /notarydir/fixtures &&
+          notary-server -config=/notarydir/fixtures/server-config-local.json"
+
+      notarysigner:
+        image: notary:signer-0.7.0
+        networks:
+          - sandbox
+        volumes:
+          - ./notary:/notarydir
+        command: |-
+          sh -c "/notarydir/migrations/migrate.sh &&
+          cd /notarydir/fixtures &&
+          notary-signer -config=/notarydir/fixtures/signer-config-local.json"
+
+      trustsandbox:
+        image: docker:20.10.20-dind
+        networks:
+          - sandbox
         volumes:
-          notarycerts:
-            external: false
+          - ./notary:/notary
+        privileged: true
+        entrypoint: ""
+        command: |-
+            sh -c '
+                cp /notary/fixtures/root-ca.crt /usr/local/share/ca-certificates/root-ca.crt &&
+                update-ca-certificates &&
+                dockerd-entrypoint.sh --insecure-registry sandboxregistry:5000'
+
+      sandboxregistry:
+        image: registry:2.8.1
```

Comment:
Might be okay here to use `registry:2` to get the latest minor+patch release of registry v2. For some cases, pinning to a "patch" release, but in this case we just need a registry for testing.